interfaces/builtin/system_observe.go

```diff
@@ -51,11 +51,15 @@ deny ptrace (trace),
 @{PROC}/vmstat r,
 @{PROC}/diskstats r,
 @{PROC}/kallsyms r,
+@{PROC}/meminfo r,
+@{PROC}/sys/kernel/osrelease r,
 
 # These are not process-specific (/proc/*/... and /proc/*/task/*/...)
 @{PROC}/*/{,task/,task/*/} r,
 @{PROC}/*/{,task/*/}auxv r,
 @{PROC}/*/{,task/*/}cmdline r,
+@{PROC}/*/{,task/*/}exe r,
+@{PROC}/*/{,task/*/}smaps r,
```

Comment:
The bug states that /proc/<pid>/smaps is needed but this is already part of the default template. I suggest updating the default template to use your rule here with 'task/' instead.